spec/spec_helper.rb

```diff
@@ -21,6 +21,10 @@
 
 RSpec.configure do |config|
   # We use capybara with webkit, and need database_cleaner
+  config.before(:suite) do
+    DatabaseCleaner.clean_with(:truncation)
+  end
```

Comment:
Just out of curiosity: Why did you add this? (Tests seem to run fine without it. Could it be this has accidentally slipped in from some other change?)